client/src/store/index.ts

```diff
@@ -387,13 +388,13 @@ const {
       state.scanDatasets = {};
       state.datasets = {};
     },
-    setCurrentImageId(state, imageId) {
-      state.currentDatasetId = imageId;
+    setCurrentFrameId(state, frameId) {
+      state.currentDatasetId = frameId;
     },
-    setImage(state, { imageId, image }) {
+    setFrame(state, { frameId, frame }) {
       // Replace with a new object to trigger a Vuex update
       state.datasets = { ...state.datasets };
-      state.datasets[imageId] = image;
+      state.datasets[frameId] = frame;
```

Comment:
In the spirit of renaming things, should this object be called `frames` rather than `datasets`, since that's what it is storing?

Reply:
Good Idea; adding to commit.